index.html

```diff
@@ -64,6 +64,201 @@ <h1>I’m Hussen. Glad to see you!</h1>
+                <h3><b>CANOPY</b></h3>
+                <ol class="frames-list">
+                  <li>Back End Dev</li>
+                  <li>2015</li>
+                </ol>
+              </div>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="tag-list">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+              <button class="blocks-button">See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="about-section">
+        <div class="about-Container">
+          <div class="text-1">
+            <h1>About Myself</h1>
+          </div>
+          <div class="text-2">
+            <p>
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website. Look through some of my work and experience!
+              If you like what you see and have a project you need coded, don’t
+              hesitate to contact me.
+            </p>
+          </div>
+
+          <div class="lets-connect">
+            <a href="#">LET’S CONNECT</a>
+          </div>
+          <ul class="social-media-icons">
+            <li class="list">
+              <a href="https://github.com/HussenYesufAlli">
+                <img src="img/g.png" alt="github" />
+              </a>
+            </li>
+            <li class="list">
+              <a href="">
+                <img src="img/Linkedin.png" alt="linkedin" />
+              </a>
+            </li>
+
+            <li class="list">
+              <a href=""> <img src="img/f.png" alt="f" /> </a>
+            </li>
+            <li class="list">
+              <a href="https://twitter.com/">
+                <img src="img/t.png" alt="tiwetter" />
+              </a>
+            </li>
+            <li class="list">
+              <a href=""> <img src="img/m.png" alt="medium" /> </a>
+            </li>
+          </ul>
+          <button class="blocks-button">Get my resume</button>
+        </div>
+
+        <ul class="skil-List">
+          <li>
+            <div class="nameList">
+              <span>Language</span><img src="img/about-img/Union.png" />
+            </div>
+            <ul class="skil-List-List">
+              <li>
+                <div class="skil-List-List-values">
+                  <img src="img/about-img/Ellipse 1.png" /><span
+                    >javaScript</span
+                  >
+                </div>
+              </li>
+              <li>
+                <div class="skil-List-List-values">
+                  <img src="img/about-img/Ellipse 2.png" /><span>HTML</span>
+                </div>
+              </li>
+              <li>
+                <div class="skil-List-List-values">
+                  <img src="img/about-img/Ellipse 3.png" /><span>CSS</span>
```

Comment:
- [ ] For this section, please do the following:
- The margin between the `languages` and javascript needs a little adjustment
- The direction of the arrows needs to be adjusted. check the images below
- The margin on both sides of this section needs to also be adjusted just as the Figma.

## Your view:
![image](https://user-images.githubusercontent.com/50990925/227729322-d3a2ea7d-6193-4f95-8d7f-8e3845200b0a.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/227729341-71bc2e82-b71e-485a-a439-5e4142304ad8.png)